python/lsst/sims/GalSimInterface/galSimPSF.py

```diff
@@ -155,7 +156,49 @@ def __init__(self, fwhm=0.6):
         #for r at half the maximum of the PSF
         alpha = fwhm/2.3835
 
-        gaussian1 = galsim.Gaussian(sigma=alpha)
-        gaussian2 = galsim.Gaussian(sigma=2.0*alpha)
+        rms_pixel_size = pixel_scale*pixel_scale/6.0
```

Comment:
Here is a python session with GalSIm confirming that convolving by a pixel adds 1/12 scale^2 to sigma^2.  Not 1/6.
```
>>> im1 = galsim.ImageD(256,256,scale=0.1)
>>> im2 = im1.copy()
>>> gaus = galsim.Gaussian(sigma=1.7)
>>> gaus_with_pixel = galsim.Convolve(gaus, galsim.Pixel(scale=1))
>>> im1 = gaus.drawImage(im1, method='no_pixel')
>>> im2 = gaus_with_pixel.drawImage(im2, method='no_pixel')
>>> sig1 = im1.FindAdaptiveMom().moments_sigma * 0.1
>>> sig2 = im2.FindAdaptiveMom().moments_sigma * 0.1
>>> sig1
1.7000000000000002
>>> sig2
1.724542808532715
>>> math.sqrt(sig2**2 - 1./12)
1.7002101532247627
>>> 1/(sig2**2 - sig1**2)
11.897977442627738
```